src/main/java/it/gov/pagopa/pu/sil/dto/PaymentsProcessingStatusDTO.java

```diff
@@ -0,0 +1,17 @@
+package it.gov.pagopa.pu.sil.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder(toBuilder = true)
+@NoArgsConstructor
+@AllArgsConstructor
+public class PaymentsProcessingStatusDTO {
+  protected String status;
```

Comment:
could be useful use the IngestionFlowFileStatusEnum here?

Reply:
the legacy value could be a detail of the SOAP exposition...
on the new API we could use directly our values